felix/rules/static.go

```diff
@@ -1067,10 +1067,11 @@ func (r *DefaultRuleRenderer) StaticManglePostroutingChain(ipVersion uint8) *gen
 
 	ipConf := r.ipSetConfig(ipVersion)
 	allIPsSetName := ipConf.NameForMainIPSet(IPSetIDAllPools)
+	allHostsSetName := ipConf.NameForMainIPSet(IPSetIDAllHostNets)
 	rules = append(rules, generictables.Rule{
 		Match: r.NewMatch().
-			SourceIPSet(allIPsSetName).
-			NotDestIPSet(allIPsSetName),
+			NotDestIPSet(allIPsSetName).
+			NotDestIPSet(allHostsSetName),
```

Comment:
Had to make this change to be able to check for packets destined for outside of cluster, by checking that destination is not in any of IP pools, nor in any of hosts.

Reply:
I guess the question is, do we want to apply DSCP to those packet paths? If we do, then what you have here is fine! If we don't, then maybe we need to tweak.

I suppose we could just maintain additional IP sets? Or we could it with multiple rules.

Reply:
ahh right. ok.